src/components/IstioWizards/WizardActions.ts

```diff
@@ -709,21 +709,21 @@ export const getInitWeights = (
           });
         }
       });
-    }
 
-    // Convention: we place the mirror routes as last position
-    if ((route as HTTPRoute).mirror) {
```

Comment:
Extra attention here: this is not an Iter8 extension change. I believe this was meant to go inside the if statement. I got an error because it was not.

Reply:
Right. I guess from the Service Wizards there is no case to get that situation, with a undefined route, which is what it should generated the error, right ?

The change seems ok to me.